src/main/host/syscall/ioctl.c

```diff
@@ -227,41 +226,6 @@ static int _syscallhandler_ioctlUDPHelper(SysCallHandler* sys, UDP* udp, int fd,
     return result;
 }
 
-static int _syscallhandler_ioctlEventDHelper(SysCallHandler* sys, EventD* eventd, int fd,
-                                             unsigned long request, PluginPtr argPtr) {
-    int result = -EINVAL;
-
-    switch (request) {
-        case FIONBIO: {
```

Comment:
It doesn't look like the `FIONBIO` case was copied over to the new `eventfd.ioctl` function. Is that a mistake?

Reply:
I put this in the syscall handler (`syscall/ioctl.rs`) directly since it should apply to all file types, but maybe for consistency it would be better to copy it into each individual file (`descriptor/pipe.rs` and `descriptor/eventfd.rs`).

Reply:
Oh, I think I just missed it then. Sorry!

I think we would only want to copy to individual files if the implementation changes depending on the type?

Reply:
Agreed, I'll leave this in the syscall handler.